_episodes/debug.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Debugging Workflows"
+title: "5. Debugging Workflows"
```

Comment:
I've just noticed we should rename this episode file - to `05-debug.md` - could you do this?